pymc3/data.py

```diff
@@ -22,13 +22,15 @@
 import pymc3 as pm
 import theano.tensor as tt
 import theano
+import pandas as pd
```

Comment:
I guess this makes pandas a dependecy, doesn't it?

Reply:
it already is a dependency